lib/utils.js

```diff
@@ -1,52 +1,122 @@
-'use strict';
+'use strict'
 
-var Cache = require('./cache');
-var Mustache = require('mustache');
-var Needle = require('needle');
-var Winston = require('winston');
+const Cache = require('./cache')
+const Mustache = require('mustache')
+const Needle = require('needle')
+const Winston = require('winston')
+const CronJob = require('cron').CronJob
+const Routes = require('./routes')
+const URL = require('url')
+const removeRoute = require('express-remove-route')
```

Comment:
@napiergit This comment hasn't been addressed. Have you decided not to change it? Or did you just miss this?